src/main/resources/application.yaml

```diff
@@ -15,6 +15,102 @@ spring:
   application:
     name: Home Office Integration API
 
+ia_system_user: ${IA_SYSTEM_USERNAME:system-user}
+ia_system_user_password: ${IA_SYSTEM_PASSWORD:system-password}
+
+security:
+  anonymousPaths:
+    - "/"
+    - "/health"
+    - "/health/liveness"
+    - "/loggers/**"
+    - "/swagger-ui.html"
+    - "/swagger-resources/**"
+    - "/v2/api-docs"
+    - "/webjars/springfox-swagger-ui/**"
+    - "/favicon.ico"
+  authorisedServices:
+    - "ccd"
+    - "ccd_data"
+    - "ccd_gw"
+    - "ccd_ps"
+    - "iac"
+  authorisedRoles:
+    - "caseworker-ia"
+    - "citizen"
+  roleEventAccess:
+    citizen:
+      - "startAppeal"
+      - "editAppeal"
+      - "submitAppeal"
+      - "submitReasonsForAppeal"
+    caseworker-ia-legalrep-solicitor:
+      - "startAppeal"
```

Comment:
again we don't need all these events.  We only care about events that will trigger the home-office api which is likely to be submitAppeal.  Keep it lean for now, we can add any others we need later easily

Reply:
Removed all other events except submitAppeal